pipeline/src/pipeline-config/model/pipelineTransformedData.ts

```diff
@@ -0,0 +1,59 @@
+import { validators } from '@jvalue/node-dry-basics'
+
+export interface PipelineTransformedData {
+  id: number
+  pipelineId: number
+  healthStatus: string
+  data: unknown
+  schema: object
+}
+
+export interface PipelineTransformedDataDTO {
```

Comment:
You can use a Utility Type: `type PipelineTransformedDataDTO = Omit<PipelineTransformedData, "id">